conftest.py

```diff
@@ -0,0 +1,18 @@
+import os
+import sys
+import pytest
+import configparser
+
+# Add the root directory to the Python path
+root_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Use the pathlib module - it's more convenient.
But I'm a little bit confused - why are you doing this?